yeezyCopper.js

```diff
@@ -1,6 +1,8 @@
 // EDIT THE STUFF BELOW IN ORDER TO MAKE THIS THING WORK. :)
 // I PUT NOTES ON THE RIGHT OF EACH FIELD TO HELP YOU OUT.
 
+var shipfast = true; //IF YOU WANT OVERNIGHT SHIPPING SET TO TRUE.
```

Comment:
<img width="518" alt="screen shot 2016-06-15 at 9 11 58 pm" src="https://cloud.githubusercontent.com/assets/5168320/16102470/d46e9092-333d-11e6-80ad-7890a00d6077.png">

That makes complete sense -- Fixed this now, i put it inside the config object.